### DISCUSSION TITLE
Issue with max_line_width parameter in openai whisper client

### DISCUSSION THREAD

**Discussion by fedigiorgio** [openai/whisper#1652] [Q&A]
I'm currently using the Whisper client from OpenAI python, and I'm trying to limit the number of characters in the output by using the max_line_width parameter. However, it doesn't seem to have any effect. Could you please verify if I'm using the parameters correctly?

`openai.Audio.transcribe_raw('whisper-1', audio, f'{uuid.uuid4()}.mp3', response_format='vtt',
                            language='en', max_line_width=32)
`

I expect the max_line_width parameter to limit the width of the output lines to 32 characters, but it doesn't seem to be working as expected. Any guidance or assistance in resolving this issue would be greatly appreciated.

**Comment by glangford** [comment]
*This* version of whisper on github has a `max_line_width` parameter, but I don't see that option documented in the online service hosted by OpenAI. Worth checking over there, it is different than github whisper.

https://platform.openai.com/docs/api-reference/audio

**Comment by helliojr** [comment]
I also had the same issue, and in case anyone is still experiencing it, this solved it for me:

"To use your word_option make sure you are using word_timestamps=True & task='transcribe' flags in DecodingOptions() or transcribe()."
 
```
import whisper
from whisper.utils import get_writer

model = whisper.load_model('base.en')

whisper.DecodingOptions(language='en', fp16='false', word_timestamps=True, task='transcribe')
audio = 'final_video.mp4'
result = model.transcribe(audio)
output_directory = "./"
word_options = {
"highlight_words": True,
"max_line_count": 50,
"max_line_width": 3
}

srt_writer = get_writer("srt", output_directory)
srt_writer(result, audio, word_options)`
```

Credits for the answer:
https://stackoverflow.com/a/77770364
